tests/datastore/providers/azuresearch/test_azuresearch_datastore.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+import os
+import time
+from typing import Union
+from azure.search.documents.indexes import SearchIndexClient
+from models.models import DocumentMetadataFilter, Query, Source, Document, DocumentMetadata
+
+AZURESEARCH_TEST_INDEX = "testindex"
+os.environ["AZURESEARCH_INDEX"] = AZURESEARCH_TEST_INDEX
```

Comment:
There's no default that will actually make tests work for all cases without provisioning your own search service, you need to indicate a test search service we can hit. Are you suggesting that we set a dummy value only so that loading the module succeeds? Or maybe I'm missing how to make this not disrupt the rest of the tests.

Reply:
exactly, it's so the module loads. when you open this up in vscode, it fails to find all of the tests because of azuresearch failing to load :)

Reply:
Done. Now other datastores fail, but it's progress :)